### PR TITLE
stable build.gradle

### PR DESCRIPTION
**Pull request by TacoTheDank**

### CODE REVIEW

**Comment by Adonai** [comment]
also, @adrian-bl , I'm really tired of asking this but could you please enable Travis integration on travis.org? .travis.yml file is there for years, you could use it for nightlies easily

**Comment by TacoTheDank** [comment]
lol why would they have the .travis.yml and not even use travis-ci? thats so pointless

File: app/build.gradle [hunk 1]
```diff
@@ -27,6 +27,6 @@ android {
 }
 
 dependencies {
-    compile 'com.android.support:support-core-ui:27.1.1'
+    implementation 'com.android.support:support-core-ui:27.1.1'
```
Comment (adrian-bl) [inline app/build.gradle]: why do we need that?

**Comment by adrian-bl** [comment]
@Adonai ok, so i clicked around and added vanilla music - travis fails with:

```
File /home/travis/.android/repositories.cfg could not be loaded.
Checking the license for package Android SDK Build-Tools 26.0.2 in /usr/local/android-sdk/licenses
Warning: License for package Android SDK Build-Tools 26.0.2 not accepted.
FAILURE: Build failed with an exception.
* What went wrong:
A problem occurred configuring project ':app'.
> You have not accepted the license agreements of the following SDK components:
  [Android SDK Build-Tools 26.0.2].
  Before building your project, you need to accept the license agreements and complete the installation of the missing components using the Android Studio SDK Manager.
  Alternatively, to learn how to transfer the license agreements from one workstation to another, go to http://d.android.com/r/studio-ui/export-licenses.html
```

i'm underwhelmed.
Any idea why it even tries to mess with 26.0.2? I don't see any reference to it in our tree.

File: app/build.gradle (hunk 1 above)
Comment (TacoTheDank) [reply app/build.gradle]: compile is being deprecated

**Comment by adrian-bl** [comment]
So i downgraded build-tools from 27.0.2 to 26.0.2 (the version travis wanted) and the build now passes.

**Pull request merged**